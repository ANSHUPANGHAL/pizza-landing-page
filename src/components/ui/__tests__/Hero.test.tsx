import { render } from '@testing-library/react';
import Home from '@/app/(home)/page';

test('Hero Section matches snapshot', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
