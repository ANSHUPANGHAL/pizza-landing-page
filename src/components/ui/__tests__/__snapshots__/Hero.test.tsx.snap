// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Section matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="bg-white"
  >
    <div
      class="container flex flex-col md:flex-row items-center justify-between py-24 px-6"
    >
      <div
        class="text-center md:text-left"
        style="opacity: 0; transform: translateX(-50px);"
      >
        <h1
          class="text-6xl md:text-7xl font-black font-sans leading-tight"
          style="opacity: 0; transform: translateY(-20px);"
        >
          Super Delicious Pizza in 
          <br />
          <span
            class="text-primary"
          >
            Only 45 Minutes!
          </span>
        </h1>
        <p
          class="text-2xl mt-6 max-w-lg leading-snug"
        >
          Enjoy a Free Meal if Your Order Takes More Than 45 Minutes!
        </p>
        <div
          style="opacity: 0; transform: scale(0.9);"
        >
          <button
            class="inline-flex items-center justify-center whitespace-nowrap ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 text-primary-foreground h-10 mt-8 text-lg rounded-full py-6 px-8 font-bold bg-orange-500 hover:bg-orange-600 transition-all duration-300"
          >
            Get your pizza now
          </button>
        </div>
      </div>
      <div
        style="opacity: 0; transform: translateX(50px);"
      >
        <img
          alt="pizza-main"
          class="max-w-full"
          data-nimg="1"
          decoding="async"
          fetchpriority="high"
          height="400"
          src="/_next/image?url=%2Fpizza-main.png&w=828&q=75"
          srcset="/_next/image?url=%2Fpizza-main.png&w=640&q=75 1x, /_next/image?url=%2Fpizza-main.png&w=828&q=75 2x"
          style="color: transparent;"
          width="400"
        />
      </div>
    </div>
  </section>
  <section>
    <div
      class="container py-12"
    >
      <div
        data-orientation="horizontal"
        dir="ltr"
      >
        <div
          aria-orientation="horizontal"
          class="inline-flex h-10 items-center justify-center rounded-md bg-muted p-1 text-muted-foreground"
          data-orientation="horizontal"
          role="tablist"
          style="outline: none;"
          tabindex="0"
        >
          <button
            aria-controls="radix-:r0:-content-pizza"
            aria-selected="true"
            class="inline-flex items-center justify-center whitespace-nowrap rounded-sm px-3 py-1.5 font-medium ring-offset-background transition-all focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 data-[state=active]:bg-background data-[state=active]:text-foreground data-[state=active]:shadow-sm text-md"
            data-orientation="horizontal"
            data-radix-collection-item=""
            data-state="active"
            id="radix-:r0:-trigger-pizza"
            role="tab"
            tabindex="-1"
            type="button"
          >
            Pizza
          </button>
          <button
            aria-controls="radix-:r0:-content-beverages"
            aria-selected="false"
            class="inline-flex items-center justify-center whitespace-nowrap rounded-sm px-3 py-1.5 font-medium ring-offset-background transition-all focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 data-[state=active]:bg-background data-[state=active]:text-foreground data-[state=active]:shadow-sm text-md"
            data-orientation="horizontal"
            data-radix-collection-item=""
            data-state="inactive"
            id="radix-:r0:-trigger-beverages"
            role="tab"
            tabindex="-1"
            type="button"
          >
            Beverages
          </button>
        </div>
        <div
          aria-labelledby="radix-:r0:-trigger-pizza"
          class="mt-2 ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2"
          data-orientation="horizontal"
          data-state="active"
          id="radix-:r0:-content-pizza"
          role="tabpanel"
          style="animation-duration: 0s;"
          tabindex="0"
        >
          <div
            class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-6 mt-6"
            style="opacity: 0;"
          >
            <div
              class="border bg-card text-card-foreground shadow-sm border-none rounded-xl"
            >
              <div
                class="flex-col space-y-1.5 p-6 flex items-center justify-center"
              >
                <img
                  alt="pizza-image"
                  data-nimg="1"
                  decoding="async"
                  height="150"
                  loading="lazy"
                  src="/_next/image?url=%2Fpizza-main.png&w=384&q=75"
                  srcset="/_next/image?url=%2Fpizza-main.png&w=256&q=75 1x, /_next/image?url=%2Fpizza-main.png&w=384&q=75 2x"
                  style="color: transparent;"
                  width="150"
                />
              </div>
              <div
                class="p-6 pt-0"
              >
                <h2
                  class="text-xl font-bold"
                >
                  Margarita Pizza
                </h2>
                <p
                  class="mt-2"
                >
                  This is a very tasty pizza
                </p>
              </div>
              <div
                class="p-6 pt-0 flex items-center justify-between mt-4"
              >
                <p>
                  <span>
                    From 
                  </span>
                  <span
                    class="font-bold"
                  >
                    ₹500
                  </span>
                </p>
                <button
                  aria-controls="radix-:r3:"
                  aria-expanded="false"
                  aria-haspopup="dialog"
                  class="bg-orange-200 hover:bg-orange-300 text-orange-500 px-6 py-2 rounded-full shadow hover:shadow-lg outline-none focus:outline-none ease-linear transition-all duration-150"
                  data-state="closed"
                  type="button"
                >
                  Choose
                </button>
              </div>
            </div>
            <div
              class="border bg-card text-card-foreground shadow-sm border-none rounded-xl"
            >
              <div
                class="flex-col space-y-1.5 p-6 flex items-center justify-center"
              >
                <img
                  alt="pizza-image"
                  data-nimg="1"
                  decoding="async"
                  height="150"
                  loading="lazy"
                  src="/_next/image?url=%2Fpizza-main.png&w=384&q=75"
                  srcset="/_next/image?url=%2Fpizza-main.png&w=256&q=75 1x, /_next/image?url=%2Fpizza-main.png&w=384&q=75 2x"
                  style="color: transparent;"
                  width="150"
                />
              </div>
              <div
                class="p-6 pt-0"
              >
                <h2
                  class="text-xl font-bold"
                >
                  Margarita Pizza
                </h2>
                <p
                  class="mt-2"
                >
                  This is a very tasty pizza
                </p>
              </div>
              <div
                class="p-6 pt-0 flex items-center justify-between mt-4"
              >
                <p>
                  <span>
                    From 
                  </span>
                  <span
                    class="font-bold"
                  >
                    ₹500
                  </span>
                </p>
                <button
                  aria-controls="radix-:r6:"
                  aria-expanded="false"
                  aria-haspopup="dialog"
                  class="bg-orange-200 hover:bg-orange-300 text-orange-500 px-6 py-2 rounded-full shadow hover:shadow-lg outline-none focus:outline-none ease-linear transition-all duration-150"
                  data-state="closed"
                  type="button"
                >
                  Choose
                </button>
              </div>
            </div>
            <div
              class="border bg-card text-card-foreground shadow-sm border-none rounded-xl"
            >
              <div
                class="flex-col space-y-1.5 p-6 flex items-center justify-center"
              >
                <img
                  alt="pizza-image"
                  data-nimg="1"
                  decoding="async"
                  height="150"
                  loading="lazy"
                  src="/_next/image?url=%2Fpizza-main.png&w=384&q=75"
                  srcset="/_next/image?url=%2Fpizza-main.png&w=256&q=75 1x, /_next/image?url=%2Fpizza-main.png&w=384&q=75 2x"
                  style="color: transparent;"
                  width="150"
                />
              </div>
              <div
                class="p-6 pt-0"
              >
                <h2
                  class="text-xl font-bold"
                >
                  Margarita Pizza
                </h2>
                <p
                  class="mt-2"
                >
                  This is a very tasty pizza
                </p>
              </div>
              <div
                class="p-6 pt-0 flex items-center justify-between mt-4"
              >
                <p>
                  <span>
                    From 
                  </span>
                  <span
                    class="font-bold"
                  >
                    ₹500
                  </span>
                </p>
                <button
                  aria-controls="radix-:r9:"
                  aria-expanded="false"
                  aria-haspopup="dialog"
                  class="bg-orange-200 hover:bg-orange-300 text-orange-500 px-6 py-2 rounded-full shadow hover:shadow-lg outline-none focus:outline-none ease-linear transition-all duration-150"
                  data-state="closed"
                  type="button"
                >
                  Choose
                </button>
              </div>
            </div>
            <div
              class="border bg-card text-card-foreground shadow-sm border-none rounded-xl"
            >
              <div
                class="flex-col space-y-1.5 p-6 flex items-center justify-center"
              >
                <img
                  alt="pizza-image"
                  data-nimg="1"
                  decoding="async"
                  height="150"
                  loading="lazy"
                  src="/_next/image?url=%2Fpizza-main.png&w=384&q=75"
                  srcset="/_next/image?url=%2Fpizza-main.png&w=256&q=75 1x, /_next/image?url=%2Fpizza-main.png&w=384&q=75 2x"
                  style="color: transparent;"
                  width="150"
                />
              </div>
              <div
                class="p-6 pt-0"
              >
                <h2
                  class="text-xl font-bold"
                >
                  Margarita Pizza
                </h2>
                <p
                  class="mt-2"
                >
                  This is a very tasty pizza
                </p>
              </div>
              <div
                class="p-6 pt-0 flex items-center justify-between mt-4"
              >
                <p>
                  <span>
                    From 
                  </span>
                  <span
                    class="font-bold"
                  >
                    ₹500
                  </span>
                </p>
                <button
                  aria-controls="radix-:rc:"
                  aria-expanded="false"
                  aria-haspopup="dialog"
                  class="bg-orange-200 hover:bg-orange-300 text-orange-500 px-6 py-2 rounded-full shadow hover:shadow-lg outline-none focus:outline-none ease-linear transition-all duration-150"
                  data-state="closed"
                  type="button"
                >
                  Choose
                </button>
              </div>
            </div>
            <div
              class="border bg-card text-card-foreground shadow-sm border-none rounded-xl"
            >
              <div
                class="flex-col space-y-1.5 p-6 flex items-center justify-center"
              >
                <img
                  alt="pizza-image"
                  data-nimg="1"
                  decoding="async"
                  height="150"
                  loading="lazy"
                  src="/_next/image?url=%2Fpizza-main.png&w=384&q=75"
                  srcset="/_next/image?url=%2Fpizza-main.png&w=256&q=75 1x, /_next/image?url=%2Fpizza-main.png&w=384&q=75 2x"
                  style="color: transparent;"
                  width="150"
                />
              </div>
              <div
                class="p-6 pt-0"
              >
                <h2
                  class="text-xl font-bold"
                >
                  Margarita Pizza
                </h2>
                <p
                  class="mt-2"
                >
                  This is a very tasty pizza
                </p>
              </div>
              <div
                class="p-6 pt-0 flex items-center justify-between mt-4"
              >
                <p>
                  <span>
                    From 
                  </span>
                  <span
                    class="font-bold"
                  >
                    ₹500
                  </span>
                </p>
                <button
                  aria-controls="radix-:rf:"
                  aria-expanded="false"
                  aria-haspopup="dialog"
                  class="bg-orange-200 hover:bg-orange-300 text-orange-500 px-6 py-2 rounded-full shadow hover:shadow-lg outline-none focus:outline-none ease-linear transition-all duration-150"
                  data-state="closed"
                  type="button"
                >
                  Choose
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-labelledby="radix-:r0:-trigger-beverages"
          class="mt-2 ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2"
          data-orientation="horizontal"
          data-state="inactive"
          hidden=""
          id="radix-:r0:-content-beverages"
          role="tabpanel"
          tabindex="0"
        />
      </div>
    </div>
  </section>
</DocumentFragment>
`;
